.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,61 @@
+# Contributing
+
+Please read EasyPost's general [contribution guidelines](https://github.com/EasyPost/.github/blob/main/CONTRIBUTING.md)
```

Comment:
does it feel a bit awkward to have to redirect them again to read the general guidelines?

Reply:
to me i think it makes sense. company-level guidelines, repo-level guidelines